test/watcher.js

```diff
@@ -431,7 +447,8 @@ group('chokidar is installed', function (beforeEach, test, group) {
 		t.plan(2);
 
 		files = ['foo-{bar,baz}.js'];
-		api.excludePatterns = ['!*bar*'];
+		// TODO(@jamestalmage, @novemberborn): There is no way for users to actually set exclude patterns yet.
+		avaFiles.defaultExcludePatterns.returns(['!*bar*']);
```

Comment:
@jamestalmage you could add exclude patterns to `files`. We should just make sure to still use the default include patterns if `files` only contains negated patterns. That's how the watcher behaves with `source` patterns.